src/app/block/[hash]/PageClient.tsx

```diff
@@ -58,10 +59,13 @@ export default function BlockPage({ params: { hash } }: any) {
                 }
               />
               <KeyValueHorizontal label={'Mined'} value={<Timestamp ts={block.block_time} />} />
-              <KeyValueHorizontal
-                label={'Transactions'}
-                value={<Value>{block.txs.length}</Value>}
-              />
+              <KeyValueHorizontal label={'Transactions'} value={<Value>{block.tx_count}</Value>} />
+              {(block as any)?.tenure_height !== null && (
+                <KeyValueHorizontal
+                  label={'Tenure Height'}
+                  value={<Value>{(block as any)?.tenure_height}</Value>}
```

Comment:
Are we able to use the correct type here?

Reply:
done